stack/dashboard/base/builder.sh

```diff
@@ -119,6 +119,9 @@ cp /root/VERSION .
 # Add exception for wazuh plugin install
 wazuh_plugin="if (plugin.includes(\'wazuh\')) {\n    return plugin;\n  } else {\n    return \`\${LATEST_PLUGIN_BASE_URL}\/\${version}\/latest\/\${platform}\/\${arch}\/tar\/builds\/opensearch-dashboards\/plugins\/\${plugin}-\${version}.zip\`;\n  }"
 sed -i "s|return \`\${LATEST_PLUGIN_BASE_URL}\/\${version}\/latest\/\${platform}\/\${arch}\/tar\/builds\/opensearch-dashboards\/plugins\/\${plugin}-\${version}.zip\`;|$wazuh_plugin|" ./src/cli_plugin/install/settings.js
+# Changed package.json build number
+jq ".build.number=$(cat /root/VERSION | tr -d '.')" ./package.json > ./package.json.tmp
+mv ./package.json.tmp ./package.json
```

Comment:
**thought**: Could the output file of `jq` command to be the `./package.json` directly instead of creating a temporal file?

For another hand, if you keep creating the temporal file, is it included in the Wazuh dashboard package? It should not.

Reply:
Being short, the JQ command left the file in blank so it isn't possible.
The move removes the temporal file :D